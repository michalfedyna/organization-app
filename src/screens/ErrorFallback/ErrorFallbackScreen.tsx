import React from 'react';
import Layout from '@components/Layout';

const ErrorFallbackScreen = () => {
  return <Layout.Container />;
};

export default ErrorFallbackScreen;
