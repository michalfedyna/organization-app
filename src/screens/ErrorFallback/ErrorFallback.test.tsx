import React from 'react';
import {test, expect} from '@jest/globals';
import renderer from 'react-test-renderer';

import ErrorFallbackScreen from './ErrorFallbackScreen';

test('ErrorFallbackScreen renders correctly', () => {
  const tree = renderer.create(<ErrorFallbackScreen />).toJSON();
  expect(tree).toMatchSnapshot();
});
