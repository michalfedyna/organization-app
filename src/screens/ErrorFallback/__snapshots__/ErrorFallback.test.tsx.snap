// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ErrorFallbackScreen renders correctly 1`] = `
<View
  style={
    {
      "backgroundColor": "#fff",
      "flex": 1,
    }
  }
/>
`;
